blog/2024/release-updates.adoc

```diff
@@ -0,0 +1,26 @@
+:title: Backwards compatibility in Keycloak releases
+:date: 2024-08-19
+:publish: false
+:author: Stian Thorgersen
+
+There will not be a Keycloak 27.0 release, at least for some time. We are certainly not going to stop bringing new
+features and enhancements to Keycloak, but we are committed to doing so in a backwards compatible way. Hence, the
+release following Keycloak 26.0 will be 26.1 instead.
+
+This means that going forwards you will no longer need to refer to lengthy migration guides to upgrade to the latest
+and greatest release, but rather just go ahead and upgrade, and if you run into any issues open a bug and we'll make
```

Comment:
When you say "any issue", you probably mean "any regression"? In our 1:1 you went a bit further and also extended that to configuration changes, maybe you want to extend this a bit more?

```suggestion
and greatest release, but rather just go ahead and upgrade, and if you run into any regressions or changes in behavior, open a bug and we'll make
```